proto/japella/japellaapi/v1/dashboard.proto

```
@@ -1,15 +1,16 @@
~syntax = "proto3";
~
 option go_package = "./gen/protobuf";
 
 import "google/api/annotations.proto";
-import "google/api/httpbody.proto";
+
+package japella.japellaapi.v1;
```

Buf (1.47.2)

Files with package "japella.japellaapi.v1" must be within a directory "japella/japellaapi/v1" relative to root but were in directory "proto/japella/japellaapi/v1".

(PACKAGE_DIRECTORY_MATCH)